addon/services/apollo.js

```diff
@@ -86,10 +88,18 @@ export default Service.extend({
    * @return {!Object}
    * @public
    */
-  clientOptions: computed(function() {
+   clientOptions: computed(function() {
     const apiURL = this.get('apiURL');
+    const credentials = this.get('credentials');
```

Comment:
I'm a little concerned that the name `credentials` can be misconstrued. Do you think it might be improved if this was called `fetchCredentials` or `requestCredentials` everywhere in this PR?

Reply:
I think that's a good idea. I used requestCredentials